application/src/main/resources/thingsboard.yml

```diff
@@ -507,6 +507,9 @@ cache:
     resourceInfo:
       timeToLiveInMinutes: "${CACHE_SPECS_RESOURCE_INFO_TTL:1440}"
       maxSize: "${CACHE_SPECS_RESOURCE_INFO_MAX_SIZE:100000}"
+    alarmTypes:
```

Comment:
alarmType page size have to be configurable

Reply:
We use the same default PageLilnk for the first request in the UI and backend. If it's configurable and is changed, the cache for our implementations will never work.